drs-auth_client/lib/drs/auth_client/client.rb

```diff
@@ -0,0 +1,81 @@
+require "active_support/core_ext/hash/keys"
+require "active_support/core_ext/string/inflections"
+require "faraday"
+
+require "drs/auth_client/models/organisation"
+require "drs/auth_client/models/profile"
+
+module Drs
+  module AuthClient
+    class Client
+      def initialize(auth_token)
+        @auth_token = auth_token
+      end
+
+      def get(path)
+        response = conn.get do |request|
+          request.url path
+          request.headers['Authorization'] = "Bearer #{@auth_token}"
+        end
+
+        if response.status == 200
+          response.body
+        else
+          nil
+        end
```

Comment:
Yeah, at the moment error handling is not present at all. The reason is that I think we need a discussion about what interface for that do we want (exceptions or dedicated error methods). I'm happy to spend more time on this if we agree what it should be.